Plutarch/Internal/PlutusType.hs

```diff
@@ -1,7 +1,8 @@
-{-# LANGUAGE AllowAmbiguousTypes #-}
```

Comment:
I assume you'll undo this

Reply:
If I don't need it after my changes that'd be neat I guess. I'll see